package.json

```diff
@@ -11,8 +11,8 @@
   },
   "scripts": {
     "test": "npm run test:node",
-    "test:node": "nyc mocha --timeout 60000 -r ts-node/register ./test/node/**/*.ts",
-    "test:simple": "mocha --timeout 60000 -r ts-node/register",
+    "test:node": "nyc mocha --timeout 60000 -r ts-node/register ./test/node/**/*.ts --require @tensorflow/tfjs-backend-cpu",
+    "test:simple": "mocha --timeout 60000 -r ts-node/register --require @tensorflow/tfjs-backend-cpu",
```

Comment:
See https://github.com/imgcook/datacook/blob/main/package.json#L14-L15, just rebase with the latest main instead of implementing again.